backend/app/services/channels/dingtalk/handler.py

```
@@ -548,6 +898,273 @@ class ChatPayload:
+                    incoming_message=incoming_message,
+                )
+                # Start the card with a "processing" message
+                await streaming_emitter.emit_chat_start(
+                    task_id=result.task.id,
+                    subtask_id=result.assistant_subtask.id,
+                    shell_type="ClaudeCode",
+                )
+                await streaming_emitter.emit_chat_chunk(
+                    task_id=result.task.id,
+                    subtask_id=result.assistant_subtask.id,
+                    content="⏳ 任务已发送到设备，正在执行中...\n\n",
```

Ruff (0.14.14)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)